src/basics.md

```diff
@@ -1265,6 +1266,32 @@ fn main() {
     println!("Time elapsed in expensive_function() is: {:?}", duration);
 }
 ```
+[ex-convert-datetime]: #ex-convert-datetime
+<a name="ex-convert-datetime"></a>
+## Converting a local time to UTC time and vice versa
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Gets the local time and displays it using [`Local::now`] and then converts it to the UTC format using the [`DateTime`] struct. A timezone is then constructed using the [`FixedOffset`] struct and the UTC time is the converted to UTC+8 and UTC-8.
```

Comment:
- s/time is the converted/time is then converted/ 
- "using the [`DateTime`] struct" could use a linked mention about  `DateTime::from_utc`